include/tvm/task_dynamics/abstract/TaskDynamics.h

```diff
@@ -76,3 +51,32 @@ namespace abstract
 }  // namespace task_dynamics
 
 }  // namespace tvm
+
+/** This macro can be used to define the derived factory required in
+ * TaskDynamics implementation, Args are the arguments required by the derived
```

Comment:
Args -> \p Args (and idem for next macro)

Reply:
Fixed in 4142dd1a